library/cmac.c

```diff
@@ -793,6 +781,18 @@ static int cmac_test_subkeys( int verbose,
         if( ( ret = mbedtls_cipher_setkey( &ctx, key, keybits,
                                        MBEDTLS_ENCRYPT ) ) != 0 )
         {
+            /* When CMAC is implemented by an alternative implementation, or
```

Comment:
The comment is not aligned with the code anymore, please update it.

Reply:
How is this comment not aligned with the code?

Reply:
The filtering of false negatives is only done for AES-192 now not for any other reason.

Reply:
Sure. I sustain that semantically, the comment is still correct. The statement is that certain features *may* be unavailable. It doesn't say anything specifically about which features are actually skipped, since that should be abundantly clear by reading the succeeding code block. The comment is there to explain why the code block is there, and is written in a way that avoids invalidating it when other bypasses are added.

I can turn it into a phrase specific to AES-192, but I feel the generic comment suits the style more.

Reply:
Done.